app/client/widgets/FieldBuilder.ts

```diff
@@ -336,9 +336,9 @@ export class FieldBuilder extends Disposable {
   // Builds the reference type table selector. Built when the column is type reference.
   public _buildRefTableSelect() {
     const allTables = Computed.create(null, (use) =>
-                                      use(this._docModel.visibleTableIds.getObservable()).map(tableId => ({
-                                        value: tableId,
-                                        label: tableId,
+                                      use(this._docModel.visibleTables.getObservable()).map(tableRec => ({
+                                        value: use(tableRec.primaryTableId),
```

Comment:
```suggestion
                                        value: use(tableRec.tableId),
```

Functionally, both of these work the same today, but `tableId` feels like a slightly better choice. (`primaryTableId` is a variant that uses a different id for summary tables, which aren't selectable here.)